### DISCUSSION TITLE
Staggered Grouping

### DISCUSSION THREAD

**Discussion by rosskd** [coccoinomane/crabada.py#110] [Q&A] [answered]
Hi guys,

Thanks so much for this awesome bit of coding. I  have encountered one issue though, I cannot get the Group Teams / Stagger Group functions to work at all, the script simply refuses to register my teams so the commands won't run.

Please can I have an example of a .env with these functions included so that I can work out where I am going wrong.

Thanks again!

**Comment by rosskd** [comment, marked as answer]
Ok so I found out where I was going wrong with this - I had to extract the new .zip file into a fresh folder (rather than copy - replace into the existing folder) and now the GROUP_TEAMS function works fine. Now waiting to see if the STAGGER_GROUP works too :)

**Comment by coccoinomane** [comment]
Thanks @rosskd for sharing your solution!

In the future, you can avoid this problem doing a `git pull` instead of downloading the zip.

If you are on Windows, you can do that with [git bash](https://gitforwindows.org/).

Cheers,
Cocco

**Reply by rosskd** [reply]
Thanks for the advice Cocco, will use a git pull in future!

Am pleased to say that the STAGGER_GROUP is working beautifully on Swimmer now, so next step is to setup the donate command so I can thank you and the team for all your amazing work.

Also, I am running a cron job on WIndows using Task Scheduler - happy to share my method with any others who want to automate your script on a Windows machine. Please let me know how and where to do so. When I have time I am also willing to flesh out / contribute to the Windows Wiki article.

Thanks again for providing this script to the community, you guys are the best!

**Comment by coccoinomane** [comment]
Thank you @rosskd for your kind words and for your donation!

When you feel like it, please go ahead and edit the Wiki article "[🆘 Windows Tutorial [TODO]]( https://github.com/coccoinomane/crabada.py/wiki/%F0%9F%86%98-Windows-Tutorial-%5BTODO%5D)".

You should be able to edit it, let me know if you have issues 🙂

Cheers,
Cocco
